### PR TITLE
DDF-3242 Updates Twitter source to work out of the box

### PR DESCRIPTION
**Pull request by vinamartin**
#### What does this PR do?
Updates Twitter source to implement ConfiguredService interface so it will work when deployed. Currently even though the availability check works, the source will not show up as available without this change.

#### Who is reviewing it? 
@ahoffer 

#### Choose 2 committers to review/merge the PR. 
@clockard
@coyotesqrl

#### How should this be tested? (List steps with links to updated documentation)
Build and deploy the twitter source into any running DDF instance. 
Configure the source via the sources tab with an API key and secret, these can be obtained from Twitter's website. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by clockard** [comment]
build now

**Comment by bdeining** [comment]
Is there a ticket associated with this PR?

File: distribution/sdk/sample-twitter-source/src/main/java/org/codice/ddf/catalog/twitter/source/TwitterSource.java [hunk 1]
```diff
@@ -183,10 +186,9 @@ private Metacard getMetacard(Status status) {
         metacard.setSourceId(id);
         metacard.setId(String.valueOf(status.getId()));
         metacard.setTitle(status.getText());
-        metacard.setMetadata("<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>" +
-                "<Resource>" +
-                "<name>" + status.getText() + "</name>" +
-                "</Resource>");
+        metacard.setMetadata(
+                "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>" + "<Resource>"
```
Comment (coyotesqrl) [inline distribution/sdk/sample-twitter-source/src/main/java/org/codice/ddf/catalog/twitter/source/TwitterSource.java]: ✏️ I would consider making a static final const format string, or even moving it to a resource. The string would look like
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><Resource><name>%s</name></Resource>
```
and you'd set it here with `metacard.setMetadata(String.format(formatString, status.getText())`

I suggest a resource only so you can avoid all of the escapes for quotation marks.

**Comment by coyotesqrl** [comment]
This should have an associated Jira ticket and the commit message should be updated to reflect that.

**Comment by vinamartin** [comment]
build now

**Pull request merged**